src/routers/productExports.ts

```diff
@@ -1,24 +1,171 @@
-
 import createRouter from '@/utilities/functions/createRouter';
+import * as ProdExportsService from '@/services/ProductExports';
+
 import authorized from '@/middleware/authorized';
 import { UserType } from '@/models/User/UserType';
 import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/** Represents a product export in the API */
+export interface ProdExport {
+  uuid: string;
+  transportId: string;
+  datetime: string;
+  productType: string;
+  amountOfProduct: number;
+  lastModified: string;
+}
+
+/**
+ * Represents a product export creation request in the API
+ */
+export interface ProdExportReq {
```

Comment:
can you make this ProdExportCreationReq so it's clear and consistent with the other interfaces too?